src/components/auth/LogIn.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { connect } from 'react-redux';
+import { emailChanged, passwordChanged } from '../../actions';
+import { Button, AppBackgroundView } from '../common';
+import Input from './Input';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'flex-start',
+    backgroundColor: 'black',
+  },
+  facebookButtonContainer: {
+    height: 100,
+    justifyContent: 'center',
+    backgroundColor: 'navy',
+  },
+  facebookButtonStyle: {
+    marginHorizontal: 25,
+    height: 50,
+  },
+  inputContainer: {
+    flex: 1,
+    marginTop: 10,
+  },
+  loginButtonContainer: {
+    flex: 1,
+    justifyContent: 'flex-end',
+    marginBottom: 25,
+  },
+  loginButtonStyle: {
+    marginHorizontal: 20,
+    height: 50,
+  },
+});
+
+class LogIn extends Component {
+  constructor() {
+    super();
+
+    this.onEmailChange = this.onEmailChange.bind(this);
+    this.onPasswordChange = this.onPasswordChange.bind(this);
+  }
+  onEmailChange(text) {
+    this.props.emailChanged(text);
+  }
+  onPasswordChange(text) {
+    this.props.passwordChanged(text);
+  }
+  render() {
+    return (
+      <AppBackgroundView style={styles.container}>
+        <View style={styles.facebookButtonContainer}>
+          <View style={styles.facebookButtonStyle}>
+            <Button backgroundColor="#007fff" fontColor="white">
+              LOG IN WITH FACEBOOK
+            </Button>
+          </View>
+        </View>
+        <View style={styles.inputContainer}>
+          <Input
+            label="EMAIL"
+            placeholder="HELLO@WOLRD.COM"
+            value={this.props.email}
+            onChangeText={this.onEmailChange}
```

Comment:
its a better idea to store `onChange` value to component state instead of redux(app state).
u should get `value` from `this.props.email` and set it to component state if a value exist (e.g. fetching this users data from an api).

probably unlikely for this particular scenario, but for any other forms in the project ... if you grab the value for `this.props.auth.email` in a different component, that value may have changed without you knowing due to the user modifying this field.